### ISSUE TITLE
Simplify Operand Descriptions

### ISSUE THREAD

**Issue by NicholasMoser** [NicholasMoser/GNTool#95]
Operand descriptions are still a bit too wordy, one example:

```asm
or *gpr2 + offset 0x0, 0x10000
```

Consider using something more similar to what you'd see from other disassemblers:

```asm
or r2(0), 0x10000
```

**Comment by NicholasMoser** [comment]
@Athkore mentioned in https://github.com/NicholasMoser/GNTool/pull/96 trying to simplify SEQ operands like `*seq_p_sp->field_0x260`, especially in cases where we know the names. Based on your comments, here's what I see as the list of ways we could change it:

1. `seq_p_sp->field_0xC`
2. `seq_p_sp(0xC)`
3. `seq_p_sp (0xC)`
4. `0xC (seq_p_sp)`
5. `0xC(seq_p_sp)`
6. `seq_p_sp[0x3]`

And for names fields maybe something like:

1. `seq_p_sp->flags`

Do you have a preference?

**Comment by NicholasMoser** [comment]
Also worth considering how to simplify something like this:
`*seq_p_sp->field_0x38 + offset 0x23C`
which happens to be the current action id of a character

**Comment by Athkore** [comment]
To simplify *seq_p_sp->field_0x38 + offset 0x23C, the simplest would probably be to replace seq_p_sp with the known field name, and if unknown replace as s<number>(number being operand given to seq_regcmd - 0x18), or even as r<number>(number being operand given to seq_regcmd). So this specific example could be something akin to chr_p(0x23C) or chr_p->action_id (sE->field_0x23C if both unknown).

From what I can discern, all fields in seq_p_sp are 4 bytes in size, and they work pretty much like and extension to reg_p, so that they combined contain 0x3F registers, with registers 0x00-0x12 are general purpose, while registers 0x13-0x3f are all special purpose registers. Knowing that they belong to different structures don't add anything really to functionality when looking at it from the angle of writing seq codes. I still feel that keeping them as different kinds of registers would be beneficial. 

As for ordering of notation, I feel kinda similar, except for being a bit partial of treating seq_p_sp fields like registers, and keep them in the naming. I believe something like that would cut down tremendously on verbosity, while still keeping things apart at a glance.

 1. s3
 2. s3(0x0)
 3. s3->field_0x0
 4. seq_p_sp->field_0xC
 5. *seq_p_sp->field_0xC (this is what we have today)
 6. seq_p_sp(0xC)
 7. *seq_p_sp(0xC)
 8. seq_p_sp (0xC)
 9. *seq_p_sp (0xC)
 10. 0xC (seq_p_sp)
 11. 0xC(seq_p_sp)

**Comment by NicholasMoser** [comment]
> So for seq_p_sp->field_0xC, it would be seq_p_sp12.

That sounds good to me!

**Comment by Athkore** [comment]
I didn't think properly when I wrote the last one, I meant for seq_p_sp->field_0xC to be seq_p_sp3, since they're only reachable 4 byte aligned. But I can definitely work with seq_p_sp12 as well, just need to rename a bunch of stuff
